main.css

```diff
@@ -0,0 +1,94 @@
+/* @import "normalize.css"; */
+
+div {
+  font-family: 'Roboto', sans-serif;
+  font-size: 22px;
+}
+
+* {
+  box-sizing: border-box;
+  border-color: #999999;
+}
+
+.calculator-container {
+  display: flex;
+  justify-content: space-around;
+  flex-flow: row wrap;
+  position: absolute;
+  height: 320px;
+  width: 240px;
+  top: 50%;
+  left: 50%;
+  margin: -200px 0 0 -150px;
+}
+
+.calculator-display {
+  border-radius: 5px 5px 0px 0px;
+  background-color: #999999;
+  color: #ffffff;
+  display: flex;
+  flex-direction: row;
+  font-size: 48px;
+  justify-content: flex-end;
+  align-items: flex-end;
+  height: 80px;
+  width: 235px;
+}
+
+.calculator-display span {
+  margin-right: 20px;
+  margin-bottom: 5px;
+}
+
+button {
+  background: none;
+  color: #000000;
+  outline: none;
+  border: none;
+  font-size: 1em;
+  padding: 12px;
+}
+
+button:active :focus {
+  outline: none;
+  border: none;
+}
+
+.calculator-button {
```

Comment:
There are a lot of the same rules in both `.calculator-button`, and `.calculator-opbutton`.  Consider applying both styles to the elements that are operators, and specializing the `.calculator-opbutton` to only apply the styles that differ from the standard button.  As well, consider removing the abbreviation in the class name, for clarity: `calculator-operator`.